src/utils/formValidationOptions.ts

```diff
@@ -43,3 +44,18 @@ export const urlValidation: (name: string) => RegisterOptions = (name) => ({
     message: `${name} must be a valid url`,
   },
 })
+
+// Validates DD/MM/YYYY formatted dates
+export const dateValidation = (date: string, required = false) => {
+  if (!date) return !required
+
+  const reversedDate = date.split('/').reverse().join('-')
+  const parsedDate = parseISO(reversedDate)
```

Comment:
Is this really needed? I think `date-fns` should be able to parse an arbitrary date format? https://date-fns.org/v2.21.1/docs/parse